test/functional/apps/dashboard_elements/controls/options_list/options_list_dashboard_interaction.ts

```diff
@@ -218,16 +221,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('Applies options list control options to dashboard', async () => {
-        await retry.try(async () => {
-          expect(await pieChart.getPieSliceCount()).to.be(2);
-        });
+        await dashboard.waitForRenderComplete();
+        expect(await pieChart.getPieSliceCount()).to.be(2);
       });
 
       it('Applies options list control options to dashboard by default on open', async () => {
         await dashboard.gotoDashboardLandingPage();
         await header.waitUntilLoadingHasFinished();
         await dashboard.clickUnsavedChangesContinueEditing(OPTIONS_LIST_DASHBOARD_NAME);
-        await header.waitUntilLoadingHasFinished();
+        await dashboard.waitForRenderComplete();
```

Comment:
Because we are checking something about the chart, `waitForRenderComplete` (which relies on `RenderCompleteDispatcher`) is more appropriate here than `waitUntilLoadingHasFinished`